9-writing-advanced-plans/README.md

```diff
@@ -55,8 +55,9 @@ In the above plan we:
 
 * Accept a comma-separated list of nodes
 * Run the `exercise9::yesorno` task from above on all of our nodes
-* Store the results of running the task in the variable `$results`. This will contain a `Struct` containing the node names and the data parsed from the JSON response from the task
-* We filter the list of nodes into the `$subset` variable for only those that answered `true`
+* Store the results of running the task in the variable `$results`. This will contain a `ResultSet` containing a list of `Result` objects for each node and the data parsed from the JSON response from the task
+* We filter the list of results into the `$subset` variable for only those that answered `true`
```

Comment:
Not quite what happens, unless you want to use separate variables for the result of `filter` and result of `map`.

Reply:
Updated for clarity